functions/index.js

```diff
@@ -60,12 +63,8 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
     });
 });
 
-function getIdNumber() {
-    var today = new Date();
-    var date = String(String(today.getFullYear()) + (today.getMonth() + 1)) + today.getDate();
-    var time = String(String(today.getHours()) + today.getMinutes()) + today.getSeconds();
+function getTaskId() {
+    if (category == 'Business') {
 
-    var result = date + time;
-
-    return result;
+    }
```

Comment:
Task Id will be create as below given example:

Business: B1, B2, B3, ....
Development: D1, D2, D3, .....
Marketing: M1, M2, M3, ....

So Please Modify it accordingly.
To implement the counter it is difficult to get the last task number of any task, So you can create a new variable inside the db as below

Collection: "Main"
Document: "RawDetails"
Field: "totalBusinessTasks" = 0;
Field: "totalDevelopmentTasks" = 0;
Field: "totalMarketingTasks" = 0;

And every time a new task is created based on the Category the follow number will be fetched in the backend and the will be update by incrementing with one.

And this counter will also help you create the simpler Task Id.

Reply:
yeah okay